### PR TITLE
Several enhancements

### LINKED ISSUE #404: Improvements to Demo examples
The Examples in the demo could be improved a little:
 - Example "Basic Point" should get a color
 - Example "Zoom based Point" throws error for unsupported WellKnownName
- ~Example "Alternating Line" should use bigger width (e.g. 5)~ --> Leave it as is as it is a GeoServer example

### LINKED ISSUE #407: Language Switcher should become dropdown
As there are currently five languages supported by the geostyler the button solution seems to be inappropriate. We should change this UI element to a dropdown or even a Searchcombo.

### PR DESCRIPTION
**Pull request by KaiVolland**
- add Tooltip to PreviewMap Collapse
- improve examples
- add `<LanguageSwitcher>` component

geostyler_languageswitcher

Fixes #404 
Fixes #407

### CODE REVIEW

**Review by hwbllmnn — APPROVED** [review]

File: src/LanguageSwitcher.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Select } from "antd";
+import { FC } from "react";
+import { AppLocale } from "./App";
```
Comment (jansule) [inline src/LanguageSwitcher.tsx:4]: Doesn't this lead to a circular dependency?

File: src/LanguageSwitcher.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import React from "react";
```
Comment (jansule) [inline src/LanguageSwitcher.tsx:1]:
```suggestion
import React from 'react';
```

File: src/LanguageSwitcher.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import { Select } from "antd";
```
Comment (jansule) [inline src/LanguageSwitcher.tsx:2]:
```suggestion
import { Select } from 'antd';
```

File: src/LanguageSwitcher.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+import { FC } from "react";
```
Comment (jansule) [inline src/LanguageSwitcher.tsx:3]:
```suggestion
import { FC } from 'react';
```

File: src/LanguageSwitcher.tsx (hunk 1 above)
Comment (jansule) [inline src/LanguageSwitcher.tsx:4]:
```suggestion
import { AppLocale } from './App';
```

File: src/assets/sample-styles/circle.json [hunk 5]
```diff
@@ -6,7 +6,8 @@
       "symbolizers": [
         {
           "kind": "Mark",
-          "wellKnownName": "Circle"
+          "wellKnownName": "Circle",
```
Comment (jansule) [inline src/assets/sample-styles/circle.json:9]:
```suggestion
          "wellKnownName": "circle",
```

**Review by jansule — COMMENTED** [review]
Just one thing left. Feel free to merge afterwards

**Pull request merged**